src/commands/eightball.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  prefix: '!eightball',
+  prefix: 'eightball',
```

Comment:
The `!help` command also needs to be updated

Reply:
All that needs to change is the removal of the `!` for the prefix value, correct?

Reply:
Yes.